packages/core/src/shipping/strategies/stripe-upe/stripe-upe-shipping-strategy.ts

```diff
@@ -97,6 +98,13 @@ export default class StripeUPEShippingStrategy implements ShippingStrategy {
         let appearance: StripeUPEAppearanceOptions;
         const styles = getStyles && getStyles();
 
+        const {
+            form: { getShippingAddressFields },
+            shippingAddress: { getShippingAddress },
+        } = this._store.getState();
+
+        const shippingFields = getShippingAddressFields([], '');
```

Comment:
Do we need to pass an empty `[]` and empty string here? Since this form is only on US can we limit that feature to US itself?

Reply:
@animesh1987, It was only for the US but now It's working for others
https://github.com/bigcommerce/bigcommerce/pull/50356